package.json

```diff
@@ -112,27 +112,32 @@
         "compile": "tsc -p ./ && node copyStaticFiles.js",
         "watch": "node copyStaticFiles.js && tsc -watch -p ./",
         "postinstall": "node ./node_modules/vscode/bin/install",
-        "test": "npm run compile && node ./node_modules/vscode/bin/test",
-        "unittest": "node_modules/.bin/mocha -u tdd out/unittest/*.js"
+        "pretest": "npm run compile",
```

Comment:
what is the use of this command?

Reply:
This command is used for building the project.